tests/chocolatey-tests/BundledApplications.Tests.ps1

```diff
@@ -2,7 +2,7 @@
 
 Describe 'Ensuring correct version of <Name> is installed' -Tag BundledApplications -ForEach @(
     @{ Name = 'shimgen' ; Version = '2.0.0' ; ChocolateyVersion = '2.0.0-alpha' ; IsSigned = $true }
-    @{ Name = '7z' ; Version = '21.07' ; ChocolateyVersion = '1.1.0' ; IsSigned = $false }
+    @{ Name = '7z' ; Version = '23.01' ; ChocolateyVersion = '2.3.0-alpha' ; IsSigned = $false }

```

Comment:
Shouldn't this be 2.2.1?

Reply:
Ah, yeah. That is fair.

It should actually be `2.2.1-alpha` (due to it needing to work before the release)